.github/workflows/docker.yml

```diff
@@ -5,36 +5,53 @@ on:
     # Weekly, at 03:00 on Monday UTC time
     - cron: "0 3 * * 1"
   pull_request:
-    paths-ignore:
-      - ".github/ISSUE_TEMPLATE/**"
-      - "*.md"
-      - "binder/**"
-      - "docs/**"
-      - "examples/**"
-      - ".pre-commit-config.yaml"
-      - ".readthedocs.yaml"
+    paths:
```

Comment:
I think this approach worked better for me.

Reply:
:+1: a bit messier to maintain this file, but for this repo where CI workflows are taking a long time, it is really really appreciated to not have to run them unless absolutely needed.